packages/docker/Dockerfile.bionic

```diff
@@ -84,4 +85,29 @@ RUN echo 'buildbot ALL=(ALL) NOPASSWD:ALL' >> /etc/sudoers
 
 USER buildbot
 
-ENTRYPOINT /entrypoint.sh "$BUILDMASTER" "$WORKERNAME" "$WORKERPASS"
+ENTRYPOINT /entrypoint.sh $BUILDMASTER $WORKERNAME" "$WORKERPASS"
+
+#####################################################################################################
+
+FROM base AS appimage
+
+# travis2bash
+RUN wget -O /usr/local/bin/travis2bash.sh https://git.bitmessage.org/Bitmessage/buildbot-scripts/raw/branch/master/travis2bash.sh
+RUN chmod +x /usr/local/bin/travis2bash.sh
+
+COPY . /home/builder/src
+
+WORKDIR /home/builder/src
+
+RUN python setup.py -V
+
+RUN python setup.py sdist
+
+RUN python setup.py  --command-packages=stdeb.command  bdist_deb
+
+RUN dpkg-deb -I deb_dist/pybitmessage_0.6.3.2-1_amd64.deb
+
+RUN buildscripts/appimage.sh
+
+RUN out/PyBitmessage-0.6.3.2.glibc2.15-x86_64.AppImage
```

Comment:
since we're running inside docker, we need to add `--appimage-extract-and-run` when running an appimage.